jvb/rootfs/defaults/sip-communicator.properties

```diff
@@ -1,6 +1,8 @@
 org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP={{ .Env.JICOFO_AUTH_USER }}@{{ .Env.XMPP_AUTH_DOMAIN }}/.*
 org.jitsi.videobridge.SINGLE_PORT_HARVESTER_PORT={{ .Env.JVB_PORT }}
-org.jitsi.videobridge.DISABLE_TCP_HARVESTER=true
+org.jitsi.videobridge.DISABLE_TCP_HARVESTER={{ .Env.JVB_TCP_HARVESTER_DISABLED }}
+org.jitsi.videobridge.TCP_HARVESTER_PORT={{ .Env.JVB_TCP_PORT }}
+org.jitsi.videobridge.TCP_HARVESTER_MAPPED_PORT={{ .Env.JVB_TCP_PORT }}
```

Comment:
I don't think this last one is needed. It only applies if different from the port.

Reply:
Have removed it for now.